src/main/java/model/OrderRepo.java

```diff
@@ -0,0 +1,44 @@
+package model;
+
+import java.util.*;
+
+public class OrderRepo {
+
+    private Map<Integer, Order> orders = new HashMap<>();
+
+
+    public List<Order> listOfOrders(){
+        ArrayList<Order> orderList = new ArrayList<>(orders.values());
+        return orderList;
+    }
+
+    public Order getOrder(Integer orderId){
+        return orders.get(orderId);
+    }
+
+    public Order addOrder(Order order){
+        return orders.put(order.getOrderId(), order);
```

Comment:
Als User muss man im Parameter nur eine order übergeben - die ID wird hier extrahiert - sehr bequem in der Benutzung - klasse :relieved: